Karma/Views/Home/RequestDialog.cshtml

```diff
@@ -36,15 +37,15 @@
             </div>
             <div class="row">
                 <div class="col-12 text text-center">
-                    If you haven’t found anything, create a request
+                    @localizer["createRequestPhrase"]
                 </div>
             </div>
             @*TODO: redirect to page (dialog?) to create a request*@
             <div class="row">
                 <button type="button" class="btn btn-light col">
                     <div class="row">
                         <div class="col-12 text-center font-weight-bold">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Posts" asp-action="CreateRequest">Create a request</a>
+                            @localizer["createRequest"]
```

Comment:
The link would stop working and when button is pressed it won't redirect to `/Posts/CreateRequest`.